docs/README.md

```diff
@@ -9,12 +9,21 @@ Packages are available at: https://github.com/orgs/terrafx/packages or via the N
 
 ## Table of Contents
 
+* [Benchmarks](#benchmarks)
 * [Code of Conduct](#code-of-conduct)
 * [License](#license)
 * [Contributing](#contributing)
 * [Goals](#goals)
 * [Languages and Frameworks](#languages-and-frameworks)
 
+### Benchmarks
+
+`Mimalloc.mi_malloc` takes **20%**-**50%** less time than `NativeMemory.Alloc` and
+`Marshal.AllocHGlobal` methods (allocation + deallocation).
+
+See more detailed benchmarks: [**raw**](./../benchmarks/BenchmarkResultsRaw.md),
```

Comment:
Markdown files and the like should go under `docs/benchmarks`, similarly to how I did samples on TerraFX proper: https://github.com/terrafx/terrafx/blob/main/docs/samples/Samples.md